src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedMethodSymbol.cs

```diff
@@ -348,5 +348,7 @@ internal override bool GenerateDebugInfo
         }
 
         internal override bool IsDeclaredReadOnly => UnderlyingMethod.IsDeclaredReadOnly;
+
+        internal sealed override bool IsInitOnly => UnderlyingMethod.IsInitOnly;
```

Comment:
>sealed [](start = 17, length = 6)

Sealing feels inappropriate.

Reply:
Are you proposing to remove `sealed` instead?
We can remove `sealed` if we ever have a more specific implementation. 

---
In reply to: [411007115](https://github.com/dotnet/roslyn/pull/43275#discussion_r411007115) [](ancestors = 411007115)

Reply:
> Are you proposing to remove sealed instead?

Yes.

> We can remove sealed if we ever have a more specific implementation. 

What is the point to seal it? This actually removes it from the list of valid override candidates in completion list. Making it very easy to miss to override the API properly. It is fine to provide default implementation in this type, but I think we shouldn't be sealing any of them simply because it sends the wrong message and likely to lead to an oversight when derived types are implemented. 


---
In reply to: [411548036](https://github.com/dotnet/roslyn/pull/43275#discussion_r411548036) [](ancestors = 411548036,411007115)